src/System.CommandLine.Suggest.Tests/SuggestionShellScriptHandlerTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Xunit;
+
+namespace System.CommandLine.Suggest.Tests
+{
+    public class SuggestionShellScriptHandlerTest
+    {
+        [Fact]
+        public void When_shellName_does_not_support_it_should_print_error()
+        {
+            var console = new TestConsole();
+            Action a = () => SuggestionShellScriptHandler.Handle(console, "fish");
+            a.Should().Throw<SuggestionShellScriptException>().And.Message.Should()
+                .Contain("fish shell is not supported.");
```

Comment:
This test will break when we add fish shell support.

Reply:
change to something like "nonexistshell"?

Reply:
changed to "notsupported"